boards/holybro/kakutef7/init/rc.board_defaults

```diff
@@ -12,12 +12,10 @@ param set-default CBRK_SUPPLY_CHK 894281
 # Select the Generic 250 Racer by default
 param set-default SYS_AUTOSTART 4050
 
-# use the Q attitude estimator, it works w/o mag or GPS.
-param set-default SYS_MC_EST_GROUP 3
-param set-default ATT_W_ACC 0.4
-param set-default ATT_W_GYRO_BIAS 0
-
+# use EKF2 without mag
```

Comment:
you might also want to set `EKF2_MAG_TYPE 5` (none)

Reply:
I didn't do that for the other Kakutes either. So you're saying I should?

Reply:
Nope, in fact it's fine, the EKF will still initialize the heading using the yaw estimator if there is no mag data: https://github.com/PX4/PX4-Autopilot/blob/d4c7ea74ef470930500c512d1b82bdc1c16752b0/src/modules/ekf2/EKF/gps_control.cpp#L126-L141